src/test/ui/const-generics/min_const_generics/const-argument-if-length.rs

```diff
@@ -0,0 +1,18 @@
+#![feature(min_const_generics)]
```

Comment:
I think it's better to instead use revisions here :thinking: 

we already do this for most other tests

Reply:
Yes, you're completely right: for some reason this slipped my mind. I'll fix those up soon.

Reply:
I've switched to using revisions.